content/events/2021/04/2021-04-20-social-media-spring-session-2021.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Social Media Spring Session 2021
+deck: Join us for updates in social media topics including advertising
```

Comment:
The deck is for the subtitle. It shouldn't be a copy of the summary. As an optional field, I don't think it's needed here. @Gabr1elleKING

Reply:
Just updated the PR to remove the deck  section. Thanks!